pom.xml

```diff
@@ -492,9 +398,9 @@
                     </executions>
                 </plugin>
                 <plugin>
-                    <groupId>org.netbeans.tools</groupId>
+                    <groupId>org.jboss.tools</groupId>
                     <artifactId>sigtest-maven-plugin</artifactId>
-                    <version>1.0</version>
+                    <version>2.0</version>
```

Comment:
I didn't run the signature tests, so I just want to confirm, this is in the public jboss repository? I didn't see it in maven central.

Reply:
Yes, https://repository.jboss.org/nexus/content/repositories/releases/org/jboss/tools/sigtest-maven-plugin/2.0/
but I thought this should sync to central. I do see it has not. Note that the concern that the GPL license of the original org.netbeans.tools:sigtest-maven-plugin preventing a fork of the project into the Jakarta tck-tools project seems unwarranted as the EMO has indicated it should not be a problem. The Validation TCK release should wait for that to happen.